app/src/main/java/edu/grinnell/appdev/grinnelldirectory/Interfaces/APICallerInterface.java

```diff
@@ -0,0 +1,20 @@
+package edu.grinnell.appdev.grinnelldirectory.Interfaces;
+
+import java.util.List;
+
+import edu.grinnell.appdev.grinnelldirectory.Model.Person;
+
+/**
+ * Created by nicholasroberson on 2/15/17.
+ */
+
+public interface APICallerInterface {
+
+    List<Person> onSearchSuccess(List<Person> people);
+
+    boolean authenticateUserCallSuccess(List<Person> people);
+
+    String onServerFailure(String fail_message);
+
+    boolean onNetworkingError(String fail_message);
```

Comment:
@NickRoberson all of these methods should return `void` 
Please update this and make sure the NetworkingTest passes afterwards